docs/faq.md

```diff
@@ -0,0 +1,10 @@
+# General FAQ
+
+- AMD GPUs: Changing resolution, using DRI_PRIME and doing various other things can cause crashes, but `echo high| sudo tee /sys/bus/pci/drivers/amdgpu/0000:??:??.?/power_dpm_force_performance_level` or adding the `amdgpu.dpm=0` to the kernel commandline stops these crashes.
+- MacBook fans running up, screen freezing, then it turns off: T2 panic; try a patched kernel.
```

Comment:
At least in what I've experienced, first the screen freezes, then the fans go high and it shuts off.

> try a patched kernel.

If we are adding instructions on how to deal with T2 crashes, it might be better to direct them to boot directly to macos and then check the crashlog.